src/opower/utilities/apco.py

```diff
@@ -0,0 +1,133 @@
+"""Appalachian Power Company (APCO)."""
```

Comment:
Rename here and elsewhere to "Appalachian Power (APCO)"? That's the name I see at https://en.wikipedia.org/wiki/American_Electric_Power and https://www.appalachianpower.com/